test/service.js

```diff
@@ -51,33 +51,74 @@ describe('Service', function () {
     .then(response => service.remove(response.receipt));
   });
 
-  describe.only('uploadFiles', () => {
+  describe('upload', () => {
     it('should be a function', () => {
-      expect(service.uploadFiles).to.be.a('function');
+      expect(service.upload).to.be.a('function');
     });
 
-    it('should send files when it is only one string', () => {
+    it('should send files when it is only one Readable', () => {
       const stub = sinon.stub(service, 'request');
       const fixturePath = path.join(__dirname, 'fixtures/files/hello.txt');
-      service.uploadFiles(fixturePath);
+      const formData = {
+        attachments: [fs.createReadStream(fixturePath)]
+      };
+      service.upload(formData);
       sinon.assert.calledOnce(stub);
       stub.restore();
     });
 
-    it('should send files when it is array of strings', () => {
+    it('should send files when it is Readable and(or) Buffer', () => {
       const stub = sinon.stub(service, 'request');
       const fixturePath1 = path.join(__dirname, 'fixtures/files/hello.txt');
       const fixturePath2 = path.join(__dirname, 'fixtures/files/image.jpg');
-      service.uploadFiles([fixturePath1, fixturePath2]);
+      const formData = {
+        attachments: [
+          fs.createReadStream(fixturePath1),
+          fs.createReadStream(fixturePath2),
+        ],
+        bufFile:  new Buffer('asfasfas'),
```

Comment:
Using the Buffer constructor is frowned upon and is deprecated:  https://nodejs.org/api/buffer.html#buffer_new_buffer_string_encoding
Use `Buffer.from(string)` method instead.
https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_string_encoding